xhr.bs

```diff
@@ -2031,6 +2039,19 @@ attributes initialized to false, so it is suggested that for consistency all
 {{ProgressEvent}} interface do the same.
 
 
+<h3 id=feature-policy>Feature Policy Integration</h3>
+
+<p>This specification defines a policy-controlled feature named <dfn>Synchronous
+XMLHttpRequest</dfn>.
+
+<p>The feature name for <a>Synchronous XMLHttpRequest</a> is "sync-xhr".
+
+<p>The default allowlist for Synchronous XMLHttpRequest is <code>*</code>.
+
+<p>When disabled in a document, calling send() on an XMLHttpRequest object with
+the synchronous flag set MUST cause a DOMException named
+ <code>NetworkError</code> to be thrown.
```

Comment:
Please remove this. We don't want duplicate requirements.

Reply:
Done.